### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by dkolas**
This could still use more love, but at least it's now more up to date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
-# Steamship Python Client Library
+<img src="docs/_static/Steamship-symbol-dark.png" alt="Steamship Logo" width="100"/>
 
-Steamship is the fastest way to build, ship, and use full-lifecycle language AI.
+# Steamship Python SDK
+
+[Steamship](https://steamship.com) is the fastest way to build, ship, and use full-lifecycle AI. 🚢
```
Comment (douglas-reid) [inline README.md:5]: should we put the emoji first?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,22 @@
-# Steamship Python Client Library
+<img src="docs/_static/Steamship-symbol-dark.png" alt="Steamship Logo" width="100"/>
 
-Steamship is the fastest way to build, ship, and use full-lifecycle language AI.
+# Steamship Python SDK
+
+[Steamship](https://steamship.com) is the fastest way to build, ship, and use full-lifecycle AI. 🚢 
+
+This library is for:
```
Comment (douglas-reid) [inline README.md:7]: do we want to mention agents here and above?  it might help segue into the next question.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+- Building and deploying `Packages` and `Plugins` on Steamship
+- Making client calls to a Steamship `Package` from a Python environment
+
+**Are you trying to build a Bot or Agent? Please check out our starter project and guidebook here:**
+
+[Multimodal Agent Starter](https://github.com/steamship-core/multimodal-agent-starter)
+
+[Agent Guidebook](https://www.steamship.com/learn/agent-guidebook)
+
+**Are you trying to host a Langchain agent you've written?  Check out our langchain compatibility library here:**
```
Comment (douglas-reid) [inline README.md:17]: we should probably use the LC emojis here, and be consistent about capitalization (as we should expect from their repos).

File: README.md (hunk 1 above)
Comment (dkolas) [reply README.md:5]: Done

File: README.md (hunk 3 above)
Comment (dkolas) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (dkolas) [reply README.md:7]: Better?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+
+**Are you trying to build a Bot or Agent? Please check out our starter project and guidebook here:**
+
+[Multimodal Agent Starter](https://github.com/steamship-core/multimodal-agent-starter)
+
+[Agent Guidebook](https://www.steamship.com/learn/agent-guidebook)
+
+**Are you trying to host a LangChain agent you've written?  Check out our LangChain compatibility library here:**
+
+🦜️🔗🚢  [Steamship-Langchain](https://github.com/steamship-core/steamship-langchain)
```
Comment (douglas-reid) [inline README.md:19]:
```suggestion
🚢 🦜️🔗 [Steamship-Langchain](https://github.com/steamship-core/steamship-langchain)
```

**Review by douglas-reid — APPROVED** [review]

**Pull request merged**